### PR TITLE
ISS-205571: Refine description of masking and add examples

### PR DESCRIPTION
**Pull request by zeeshan-devrev**
## Summary

This PR adds the examples of masking and unmasking, and also refines the description of masking in the DevRev SDK for React Native and Expo.
## Connected Issues

- #ISS-205571

## Craftsmanship, Integrity, and Devil’s Advocacy

- [ ] Testing: Negative test cases: null or default values, crash and fault injection tests
- [ ] Testing: Boundary conditions: rolling upgrades, denial-of-service, etc.
- [ ] Testing: Fixing a few existing — flaky or permanently-broken — test cases
- [ ] Observing: Detailed error codes so machines can understand
- [ ] Observing: Adding superior metrics for future debugging
- [ ] Observing: Tracing the hairiest pathways for field serviceability
- [ ] Training: KnowledgeOps update? So AI always works.

### CODE REVIEW

File: fern/docs/pages/sdks/mobile/react-native/features.mdx [hunk 1]
```diff
@@ -291,20 +291,64 @@ DevRev.clearSessionProperties()
```
[the remark this answers is not in the document]
Comment (zeeshan-devrev) [reply fern/docs/pages/sdks/mobile/react-native/features.mdx:291]: I haven't changed this title, but a good point by EkLine.

File: fern/docs/pages/sdks/mobile/react-native/features.mdx [hunk 2]
```diff
@@ -291,20 +291,64 @@
...
+
+  return (
+    <View ref={viewRef}>
+      {/* Insensitive content goes here */}
+    </View>
+  );
+};
+```
+
 ### Timers
```
[the remark this answers is not in the document]
Comment (zeeshan-devrev) [reply fern/docs/pages/sdks/mobile/react-native/features.mdx:352]: This suggestion can be ignored, as we’re not managing the timers. However, timers are used to calculate the round-trip time for any task.

File: fern/docs/pages/sdks/mobile/react-native/features.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (bojan) [reply fern/docs/pages/sdks/mobile/react-native/features.mdx:291]: This should have been fixed previously, if not change it to Mask.

File: fern/docs/pages/sdks/mobile/react-native/features.mdx
[the remark this answers is not in the document]
Comment (zeeshan-devrev) [reply fern/docs/pages/sdks/mobile/react-native/features.mdx:1]: @bojan These are the simplest and minimal examples added, as React Native’s masking differs from other platforms. It requires additional steps to get the reference (`ref`) using `useRef` and `findNodeHandle` and then pass it to our masking and unmasking methods. We added these steps to make it easier for customers to understand.

[the remark this answers is not in the document]
Comment (bojan) [reply fern/docs/pages/sdks/mobile/react-native/features.mdx:1]: That was the point, they are merged in the other platforms.

[the remark this answers is not in the document]
Comment (zeeshan-devrev) [reply fern/docs/pages/sdks/mobile/react-native/features.mdx:1]: I have combined the examples.

File: fern/docs/pages/sdks/mobile/react-native/features.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (bojan) [reply fern/docs/pages/sdks/mobile/react-native/features.mdx:348]: I agree.

**Review by bojan — APPROVED** [review]

File: fern/docs/pages/sdks/mobile/cordova/features.mdx [hunk 3]
```diff
@@ -161,21 +161,22 @@
...
 
-```javascript
-DevRev.unmarkSensitiveViews(tags, successCallback, errorCallback)
-```
+- Mark an element as masked:
+  ```html
+  <label class="devrev-mask">OTP: 12345</label>
+  ```
+- Mark an element as unmasked:
+  ```html
```
[the remark this answers is not in the document]
Comment (zeeshan-devrev) [reply fern/docs/pages/sdks/mobile/cordova/features.mdx:177]: Kept it the same as Android.

File: fern/docs/pages/sdks/mobile/flutter/features.mdx
[the remark this answers is not in the document]
Comment (zeeshan-devrev) [reply fern/docs/pages/sdks/mobile/flutter/features.mdx:1]: Since there is no change here, it can be ignored.

File: fern/docs/pages/sdks/mobile/cordova/features.mdx [hunk 4]
```diff
@@ -161,21 +161,22 @@
...
-```
-
-If any previously masked views need to be unmasked, you can use the following method:
+Masking elements can be achieved by using the CSS class `devrev-mask` to mask an element and `devrev-unmask` to unmask an element.
 
-```javascript
-DevRev.unmarkSensitiveViews(tags, successCallback, errorCallback)
-```
+- Mark an element as masked:
+  ```html
```
[the remark this answers is not in the document]
Comment (zeeshan-devrev) [reply fern/docs/pages/sdks/mobile/cordova/features.mdx:173]: Kept it the same as Android.

File: fern/docs/pages/sdks/mobile/cordova/features.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (zeeshan-devrev) [reply fern/docs/pages/sdks/mobile/cordova/features.mdx:177]: Kept it the same as Android.

File: fern/docs/pages/sdks/mobile/cordova/features.mdx [hunk 5]
```diff
@@ -161,21 +161,22 @@ To clear the session properties in scenarios such as user logout or when the ses
...
+  <label class="devrev-mask">OTP: 12345</label>
+  ```
+- Mark an element as unmasked:
+  ```html
+  <input type="text" placeholder="Enter Username" name="username" required class="devrev-unmask">
+  ```
```
[the remark this answers is not in the document]
Comment (zeeshan-devrev) [reply fern/docs/pages/sdks/mobile/cordova/features.mdx:179]: Kept it the same as Android.

File: fern/docs/pages/sdks/mobile/cordova/features.mdx [hunk 6]
```diff
@@ -161,21 +161,22 @@
...
```
[the remark this answers is not in the document]
Comment (zeeshan-devrev) [reply fern/docs/pages/sdks/mobile/cordova/features.mdx:179]: Kept it the same as Android.

File: fern/docs/pages/sdks/mobile/flutter/features.mdx [hunk 7]
```diff
@@ -338,6 +338,25 @@
...
+
+- Mark an element as masked:
+  ```html
+  <label class="devrev-mask">OTP: 12345</label>
+  ```
+- Mark an element as unmasked:
+  ```html
+  <input type="text" placeholder="Enter Username" name="username" required class="devrev-unmask">
+  ```
+
```
[the remark this answers is not in the document]
Comment (zeeshan-devrev) [reply fern/docs/pages/sdks/mobile/flutter/features.mdx:359]: Kept it the same as Android.

File: fern/docs/pages/sdks/mobile/react-native/features.mdx [hunk 8]
```diff
@@ -305,6 +305,59 @@
...
+export default YourComponent;
+```
+
+### Mask elements inside web views
+
+To mark elements as sensitive inside a web view (`WebView`), apply the `devrev-mask` CSS class. To unmark them, use `devrev-unmask`.
+
+- Mark an element as masked:
+  ```html
+  <label class="devrev-mask">OTP: 12345</label>
```
[the remark this answers is not in the document]
Comment (zeeshan-devrev) [reply fern/docs/pages/sdks/mobile/react-native/features.mdx:354]: Kept it the same as Android.

File: fern/docs/pages/sdks/mobile/flutter/features.mdx [hunk 9]
```diff
@@ -338,6 +338,23 @@ DevRevUnmask(
   )
 ```
 
+### Mask elements inside web views
+
+Input views such as password text fields are automatically masked in web views.
+
+While the auto-masking feature is sufficient for most situations, you can manually mark or unmark additional views as sensitive.
```
Comment (zeeshan-devrev) [inline fern/docs/pages/sdks/mobile/flutter/features.mdx:345]: This is additional but important: to tell customers about web view masking, as our Flutter SDK currently doesn't support auto masking of the Flutter views.

File: fern/docs/pages/sdks/mobile/cordova/features.mdx [hunk 10]
```diff
@@ -161,21 +161,22 @@
...
```
[the remark this answers is not in the document]
Comment (zeeshan-devrev) [reply fern/docs/pages/sdks/mobile/cordova/features.mdx:179]: Kept it the same as Android.

File: fern/docs/pages/sdks/mobile/flutter/features.mdx (hunk 9 above)
Comment (bojan) [reply fern/docs/pages/sdks/mobile/flutter/features.mdx:345]: No, the problem here is the `/`, the other sentence is the duplicate.

Comment (zeeshan-devrev) [reply fern/docs/pages/sdks/mobile/flutter/features.mdx]: Done.

**Review by bojan — APPROVED** [review]

File: fern/docs/pages/sdks/mobile/cordova/features.mdx [hunk 11]
```diff
@@ -161,21 +161,22 @@ To clear the session properties in scenarios such as user logout or when the ses
...
-DevRev.markSensitiveViews(tags, successCallback, errorCallback)
-```
-
-If any previously masked views need to be unmasked, you can use the following method:
+To mark elements as sensitive inside a web view (`WebView`), apply the `devrev-mask` CSS class. To unmark them, use `devrev-unmask`.
 
-```javascript
-DevRev.unmarkSensitiveViews(tags, successCallback, errorCallback)
-```
+- Mark an element as masked:
```
[the remark this answers is not in the document]
Comment (zeeshan-devrev) [reply fern/docs/pages/sdks/mobile/cordova/features.mdx:172]: Kept it the same as Android.

File: fern/docs/pages/sdks/mobile/react-native/features.mdx [hunk 12]
```diff
@@ -305,6 +305,59 @@
...
+  ```
+- Mark an element as unmasked:
+  ```html
```
[the remark this answers is not in the document]
Comment (zeeshan-devrev) [reply fern/docs/pages/sdks/mobile/react-native/features.mdx:357]: Kept it the same as Android.
